site/src/components/charts/lineChart.js

```diff
@@ -0,0 +1,89 @@
+import { Axis, Chart, LineAdvance, Tooltip } from "bizcharts";
+import styled from "styled-components";
+import moment from "moment";
+import { useEffect, useRef, useState } from "react";
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  svg,
+  img {
+    margin-top: 17px;
+  }
+`;
+
+const Title = styled.h2`
+  margin: 0;
+  font-size: 12px;
```

Comment:
plz consider using pre defined css in `/styles/text.js`
it has `font-size` `weight` `line-height` `font-family` all bundled toghter